src/pages-and-resources/discussions/DiscussionAppList.jsx

```diff
@@ -1,71 +1,47 @@
 import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
 import { Button, Container, Row } from '@edx/paragon';
+import { useDispatch, useSelector } from 'react-redux';
+
 import messages from './messages';
 import DiscussionAppCard from './DiscussionAppCard';
 import FeaturesTable from './FeaturesTable';
+import { useModels } from '../../generic/model-store';
+import { fetchApps } from './data/thunks';
 
-// XXX this is just for testing and should be removed ASAP
-const forums = [
-  {
-    forumId: 'edX Forum',
-    logo: 'https://cdn-blog.lawrencemcdaniel.com/wp-content/uploads/2018/01/22125436/edx-logo.png',
-    description: 'Start conversations with other learners, ask questions, and interact with other learners in the course.',
-    supportLevel: 'Full support',
-    isAvailable: true,
-    features: ['LTI Integration', 'Discussion Page', 'Embedded Course Sections', 'Embedded Course Units', 'WCAG 2.1 Support'],
-  },
-  {
-    forumId: 'Piazza',
-    logo: 'https://cdn-blog.lawrencemcdaniel.com/wp-content/uploads/2018/01/22125436/edx-logo.png',
-    description: 'Piazza is designed to connect students, TAs, and professors so every student can get the help they need when they need it',
-    supportLevel: 'Partial support',
-    isAvailable: true,
-    features: ['LTI Integration', 'Discussion Page', 'Embedded Course Sections', 'WCAG 2.1 Support'],
-  },
-  {
-    forumId: 'Yellowdig',
-    logo: 'https://cdn-blog.lawrencemcdaniel.com/wp-content/uploads/2018/01/22125436/edx-logo.png',
-    description: 'Yellowdig is the digital solution that impacts the entire student lifecycle and enables lifelong learning.',
-    supportLevel: 'Coming soon',
-    isAvailable: false,
-    features: ['LTI Integration', 'Discussion Page', 'Embedded Course Sections', 'WCAG 2.1 Support'],
-  },
-  {
-    forumId: 'Untitled Forum',
-    logo: 'https://cdn-blog.lawrencemcdaniel.com/wp-content/uploads/2018/01/22125436/edx-logo.png',
-    description: 'Start conversations with other learners, ask questions, and interact with other learners in the course.',
-    supportLevel: 'Full support',
-    isAvailable: true,
-    features: ['LTI Integration', 'Discussion Page', 'Embedded Course Sections', 'Embedded Course Units', 'WCAG 2.1 Support'],
-  },
-];
+function DiscussionAppList({ courseId, intl }) {
+  const [selectedAppId, setSelectedAppId] = useState(null);
 
-const featuresList = ['LTI Integration', 'Discussion Page', 'Embedded Course Sections', 'Embedded Course Units', 'WCAG 2.1 Support'];
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchApps(courseId));
+  }, [courseId]);
 
-function DiscussionAppList({ intl }) {
-  const [selectedForumId, setSelectedForumId] = useState(null);
+  const appIds = useSelector(state => state.discussions.appIds);
+  const featureIds = useSelector(state => state.discussions.featureIds);
+  const apps = useModels('apps', appIds);
+  const features = useModels('features', featureIds);
```

Comment:
Note that "features" is now a first-class model, rather than just a descriptive string, so it gets the same treatment, and the apps each have a list of feature IDs that they support.